packages/forms/src/UIForm/doc/triggers.md

```diff
@@ -0,0 +1,181 @@
+# Triggers
+
+Triggers are advanced ways to controle properties and errors.
+
+*Warning*
+Use only triggers if you can't perform what you want with another feature listed in the [documentation](../README.md).
+
+## What are triggers ?
+
+Triggers are asynchronous calls that you can perform in your custom widgets.
+
+You can do whatever you want outside of the forms, such as
+- getting info from outside, REST calls, etc
+- perform advanced validation
+- change value in another widget
+- ...
+
+## TD;LR
+
+To create a trigger you need
+- (widget) to define events that will run the trigger
```

Comment:
trigger can be called by any widget. I don t understand the point here

Reply:
Depending on the widget implementation, it defines a set of triggers that will be called.
For example, a widget will trigger on blur, but another won't.

Here I try to explain that there are
- a set of trigger events a widget supports
- a set of actions you implement in you app
- a schema definition that links the 2 sets